src/CMake/pkgconfig.cmake

```diff
@@ -12,7 +12,7 @@ else ()
 endif ()
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`